testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/log/StatefulBean.java

```diff
@@ -0,0 +1,33 @@
+package org.jboss.as.test.integration.ejb.log;
```

Comment:
Please add the copyright header to the java files.

Reply:
Copyright headers added.